### ISSUE TITLE
doc: reference/devices.md missing link

### ISSUE THREAD

**Issue by kian99** [canonical/lxd#11937]
The generated page for device.md at https://documentation.ubuntu.com/lxd/en/latest/reference/devices/#devices is missing a ref. The offending reference is below
>(see {ref}instances-configure-devices)

This looks like it was present in the old lxd docs at https://linuxcontainers.org/lxd/docs/master/howto/instances_configure/#configure-devices but seems to now be merged into the single doc at [howto/instances_configure](https://github.com/canonical/lxd/tree/main/doc/howto/instances_configure.md)

---
Document: reference/devices.md

**Comment by tomponline** [comment]
@ru-fu are you able to look into this please?

**Comment by ru-fu** [comment]
Hi @kian99 - it seems to look and work as expected here:
![image](https://github.com/canonical/lxd/assets/11227796/751d80f3-3fd2-453c-8478-388eb3247339)

Can you explain a bit more what the problem is? 
Or maybe it was just a temporary glitch.

**Comment by kian99** [comment]
Hi, apologies I went to double check and you're totally right. I did a search for "configure devices" and from the search page there are no links in the body of the text as below,
![image](https://github.com/canonical/lxd/assets/46668016/ec309e72-b977-4463-b4ff-9c9b3e6cb4e6)

and then after going to the page I thought the link was simply missing because the text was highlighted but it was actually there. Sorry for the confusion!
![image](https://github.com/canonical/lxd/assets/46668016/204931d7-0f11-40cb-9a04-331c34cd56e3)
